File: Assignment25/program3.c
// Write a program to accept a string from user and toggle the case.
// Input : "marvellous"
//output : "MARVELLOUS"

#include<stdio.h>
void struprX(char *str)
{
    while(*str != '\0')
    {
        if((*str >= 'a')&&(*str <= 'z'))
        {
            *str =*str - 32;
        }

        
       else if((*str >= 'A')&&(*str <= 'Z'))
        {
            *str =*str +32;
        }
        str++;
    }
}

int main()
{
    char arr[20];
    printf("Enter the String \n");
    scanf("%[^'\n']s",arr);

    struprX(arr);
    printf("Modified string: %s",arr);
    return 0;
}

File: Assignment42/program3.java
/*//write java program and accept file name from user open that file in write mode and write some data at the end of the file.

import java.util.Scanner;
import java.io.*;  
class program3
{  
public static void main(String args[])
{  
	try
	{  
        Scanner in = new Scanner(System.in);
        System.out.println("What is the filename?");
        String input = in.nextLine();
     	FileOutputStream fout=new FileOutputStream(input,true);  
     	String s="Vaishali 628 Marvellous Infosystems";  
     	byte b[]=s.getBytes();//converting string into byte array  
     	fout.write(b);  
     	fout.close();  
     	System.out.println("success...");  
	}
	catch(Exception e){System.out.println(e);}  
}  
} 